chia/_tests/core/mempool/test_mempool_manager.py

```diff
@@ -1943,27 +1943,25 @@ async def get_coin_records(coin_ids: Collection[bytes32]) -> List[CoinRecord]:
 
 
 TEST_FILL_RATE_ITEM_COST = 144_720_020
-QUOTE_BYTE_COST = 2 * DEFAULT_CONSTANTS.COST_PER_BYTE
-QUOTE_EXECUTION_COST = 20
 
 
 @pytest.mark.anyio
 @pytest.mark.limit_consensus_modes(allowed=[ConsensusMode.HARD_FORK_2_0])
 @pytest.mark.parametrize(
     "max_block_clvm_cost, expected_block_items, expected_block_cost",
     [
-        # Here we set the block cost limit to twice the test items' cost, so we
-        # expect both test items to get included in the block.
-        # NOTE: The expected block cost is smaller than the sum of items' costs
-        # because of the spend bundle aggregation that creates the block
-        # bundle, in addition to a small block compression effect that we
-        # can't completely avoid.
-        (TEST_FILL_RATE_ITEM_COST * 2, 2, TEST_FILL_RATE_ITEM_COST * 2 - 107_980),
+        # Here we set the block cost limit to twice the test items' cost, yet
+        # we expect only one test item to get included in the block, because we
+        # account for the block overhead in the effective maximum clvm block
+        # cost limit, preventing the second item from getting in as a result.
+        # NOTE: The cost difference here is because get_conditions_from_spendbundle
+        # does not include the block overhead.
+        (TEST_FILL_RATE_ITEM_COST * 2, 1, TEST_FILL_RATE_ITEM_COST + BLOCK_OVERHEAD),
```

Comment:
it seems like the essence of this test is changed with this 2 -> 1. Is the test still testing the same thing?
Did the change to the production code cause the test to fail? (if so, do you understand why?)